schema/iea43_wra_data_model.schema.json

```diff
@@ -744,6 +744,15 @@
                     "quality",
                     "carrier_to_noise_ratio",
                     "doppler_spectral_broadening",
+                    "true_north_offset",
+                    "orientation",
```

Comment:
What is the difference between compass_direction and orientation?

Reply:
Yeah, not sure. I wonder would there be a case where there can be a different orientation than the one we consider as referenced from North? 

If we think they are effectively the same then I would lean towards keeping `orientation` over `compass_direction` as orientation is used elsewhere in the data model to refer to directions from North e.g. `boom_oreintation`, `vane_dead_band_orientation`, `device_orientation`, `orientation_from_mast_centre` and then we use `orientation_reference_id` to say whether it is reference to true north, magnetic north or grid north.

I think also `compass_direction` is confusing as it implies it might be against magnetic north since it is a "compass". And also a bit confusing with cardinal directions i.e. are we talking about north, south, east, west or actual degrees. I would lean to removing `compass_direction` and keeping `orientation`?

Reply:
@stephenholleran Some lidar units have internal compasses. Some devices report that in the time series and some report that in the header. Everything that we've been using that has orientation also references to orientation_reference_id which can be true or magnetic. I am not aware of any remote sensing device that reports true degrees. Perhaps compass_direction is better.